src/components/SiteFooter/index.js

```diff
@@ -1,21 +1,56 @@
 import { css } from '@emotion/core';
-import { Paragraph, Link } from '@octopusthink/nautilus';
+import { Paragraph, Link, heading, useTheme } from '@octopusthink/nautilus';
 import React from 'react';
 
 import config from 'data/SiteConfig';
 
 const SiteFooter = () => {
+  const theme = useTheme();
+
   return (
     <footer
       css={css`
+        width: 100%;
         margin: 0 auto;
-        max-width: ${config.contentWidth};
-        max-width: ${config.siteContentWidth};
         padding: 8rem 0 1.6rem;
         text-align: center;
+        position: relative;
+
+        &::before {
+          content: '';
+          display: inline-block;
+          background-image: url(/images/header-bg.png);
+          background-position: bottom center;
+          background-size: 100%;
+          background-repeat: no-repeat;
+          border-top: 1px solid transparent;
+          margin: 0 auto;
+          position: absolute;
+          top: 0;
+          left: 0;
+          right: 0;
+          bottom: 0;
+          transform: scaleY(-1);
+        }
       `}
     >
-      <Paragraph light noMargin small>
+      <Paragraph
+        light
+        noMargin
+        small
+        css={css`
+          ${heading.small(theme)};
+          color: #edf0ee;
```

Comment:
Should this be a variable or something instead?

Reply:
Sure should! I've fixed it.